src/main/java/com/smattme/MysqlExportService.java

```diff
@@ -338,14 +347,32 @@ private String exportToSql() throws SQLException {
         TablesResponse allTablesAndViews = MysqlBaseService.getAllTablesAndViews(database, stmt);
 
         List<String> tables = allTablesAndViews.getTables();
-        //for every table, get the table creation and data
-        // insert statement
-        for (String s: tables) {
-            try {
-                sql.append(getTableInsertStatement(s.trim()));
-                sql.append(getDataInsertStatement(s.trim()));
-            } catch (SQLException e) {
-                logger.error("Exception occurred while processing table: " + s, e);
+
+        // Retrieve specific tables from properties, if they exist
+        String specificTables = properties.containsKey(SPECIFIC_TABLES_FOR_EXPORT)
+            ? properties.getProperty(SPECIFIC_TABLES_FOR_EXPORT)
+            : "";
+
+        // Split the specific tables into a list
+        List<String> specificTableList = new ArrayList<>();
+        if (!specificTables.isEmpty()) {
+            specificTableList = Arrays.stream(specificTables.split(",")).map(String::trim).toList();
+        }

```

Comment:
```suggestion
        List<String> specificTableList = new ArrayList<>();
        if (!specificTables.isEmpty()) {
            specificTableList = Arrays.stream(specificTables.split(",")).map(String::trim).toList();
        }
```

We can replace the above snippet with a ternary operator. As in:

```
List<String> specificTableList = !specificTables.isEmpty()
                                 ? Arrays.stream(specificTables.split(",")).map(String::trim).collect(Collectors.toList())
                                 : Collections.emptyList();
```